app/assets/stylesheets/govuk_publishing_components/components/_cookie-banner.scss

```diff
@@ -69,6 +69,10 @@ $govuk-cookie-banner-text-green: #00823b;
   display: none;
   position: relative;
   padding: govuk-spacing(1);
+
+  &:focus {
+    outline: none;
```

Comment:
Worth a comment explaining why this is OK to do for future travellers.

Reply:
Explained in the commit message, but maybe not everyone is a git history fan, so I've added a comment in the file too as suggested.